### PR TITLE
docs(pills): add documentation for pills

### PR DESCRIPTION
**Pull request by technabors**
Adding documentation for pills; notes below

-Create `pills.html`
- Added to `content-areas` category; Someone check if this is correct
- Created new Zoom hero image based on the one example I was able to track down for this pattern; Someone check if this is correct
- Create pills preview-image
- Export images as 2x png.

This doc needs a lot of love in the content editing portion. Much of the writing is ambiguous and doesn't properly explain the pattern. @araiford

### CODE REVIEW

File: _component_design/pills.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+## Introduction
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to Use
+
+Pills are small, pill-shaped containers that indicate selection and can be removed. They are useful for their small size, and for the ease with which they can be removed. As selection indicators (not selectors themselves), they have a broad set of possible applications.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/pills.md]: updated.

File: _component_design/pills.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to Use
+
+Pills are small, pill-shaped containers that indicate selection and can be removed. They are useful for their small size, and for the ease with which they can be removed. As selection indicators (not selectors themselves), they have a broad set of possible applications. 
+
+### Best practices
+
+When using pills, make sure they are uniform in size, shape, and application. Pills may be modified with color if done carefully and intentionally.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/pills.md]: updated.

File: _component_design/pills.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
+---
+title: Pills
+parent: Content Areas
+secondary: Editing
+layout: component
+category: Components
+usage: >
+  Pills are small, pill-shaped containers that indicate selection and can be removed. They are useful for their small size, and for the ease with which they can be removed. As selection indicators (not selectors themselves), they have a broad set of possible applications.
```
Comment (DonNateR) [inline _component_design/pills.md]: Remove... "and can be removed." Redundant considering the next sentence says, "for the was with which they can be removed."

File: _component_design/pills.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Default layout
+
+Pills are a component with a variety of use cases, so you can throw more or less anything you want into them to indicate something has been selected: keyword search, input text search, the name of a server you’ve selected. If it can be selected, you can make a pill of it.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/pills.md]: updated.

File: _component_design/pills.md (hunk 3 above)
Comment (DonNateR) [reply _component_design/pills.md]: Remove comma between... "for their small size and for the ease."

File: _component_design/pills.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (DonNateR) [reply _component_design/pills.md]: Same thing as in the intro. Remove "...and can be removed."

File: _component_design/pills.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Default layout
+
+Pills are a component with a variety of use cases, so you can throw more or less anything you want into them to indicate something has been selected: keyword search, input text search, the name of a server you’ve selected. If it can be selected, you can make a pill of it. 
+
+A pill is different than a tag in a variety of ways, most important of which that you cannot remove a tag.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/pills.md]: updated.

File: _component_design/pills.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Style
+
+While it is conceivable that pills are larger or smaller than body font, by and large they will exist at body font.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/pills.md]: updated.

[the remark this answers is not in the document]
Comment (DonNateR) [reply _component_design/pills.md]: Is body font just a generally understood text size amongst designers? Should the pills "generally use the same size as body font?" Might be over-explaining here, but confused me a little bit.

File: _component_design/pills.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+
+While it is conceivable that pills are larger or smaller than body font, by and large they will exist at body font. 
+
+Pills consist of a container, text, and an “x” icon to indicate removal. By default, they are `gray 400` background.
+
+### Spacing
+
+Pills have 16px of space on either side for breathing room, and 8px below and beside them when grouped together. They begin just 4px beneath other components to which they are associated – use the best top spacing for your particular use case. We encourage the use of left justification for pills.
+
+If there are a lot of pills bunched together – more than 15 – consider an alternative pattern or contact the Helix team.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/pills.md]: updated

File: _component_design/pills.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### States
+
+Pills have 16px of space on either side for breathing room, and 8px below and beside them when grouped together. They begin just 4px beneath other components to which they are associated – use the best top spacing for your particular use case. We encourage the use of left justification for pills.
+
+If there are a lot of pills bunched together – more than 15 – consider an alternative pattern or contact the Helix team.
```
Comment (DonNateR) [inline _component_design/pills.md]: Rewrite: "If more than 15 pills are bunched together, consider an alternative pattern or contact the Helix team."

File: _component_design/pills.md [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+
+### Style
+
+While it is conceivable that pills are larger or smaller than body font, by and large they will exist at body font. 
+
+Pills consist of a container, text, and an “x” icon to indicate removal. By default, they are `gray 400` background.
+
+### Spacing
+
+Pills have 16px of space on either side for breathing room, and 8px below and beside them when grouped together. They begin just 4px beneath other components to which they are associated – use the best top spacing for your particular use case. We encourage the use of left justification for pills.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/pills.md]: updated.

File: _component_design/pills.md [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### States
+
+Pills have 16px of space on either side for breathing room, and 8px below and beside them when grouped together. They begin just 4px beneath other components to which they are associated – use the best top spacing for your particular use case. We encourage the use of left justification for pills.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/pills.md]: sure isn't. Updating.

File: _component_design/pills.md (hunk 3 above)
Comment (technabors) [reply _component_design/pills.md]: updated.

File: _component_design/pills.md (hunk 8 above)
Comment (technabors) [reply _component_design/pills.md]: updated.

File: _component_design/pills.md [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### States
+
+**Active:** This state indicates that a selection has been made. This is the default state for a pill.
```
Comment (nimbinatus) [inline _component_design/pills.md]: s/`has been made. This is`/`has been made and is`

File: _component_design/pills.md [hunk 12]
```diff
@@ -0,0 +1,134 @@
...
+
+**Hover:** The user may hover over the pill to see that action can be taken upon it. While nothing will occur
```
Comment (nimbinatus) [inline _component_design/pills.md]: Missing end of sentence?

File: _component_design/pills.md [hunk 13]
```diff
@@ -0,0 +1,134 @@
...
+
+**Selected:** When a filter+search completion item have been selected.
```
Comment (nimbinatus) [inline _component_design/pills.md]: `filter+search completion item`? Not sure what this means (maybe the original designer can explain?)...

Comment (technabors) [reply _component_design/pills.md]: Removed selected state as it was not a part of the graphics in the document and wasn't meant to be included.

File: _component_design/pills.md (hunk 12 above)
Comment (technabors) [reply _component_design/pills.md]: Removed the half-sentence and edited to read, "**Hover:** The hover state indicates the pill may be interacted with to remove the selection."

**Pull request merged**